src/task/style.css

```diff
@@ -213,3 +213,45 @@ footer {
 .inputBox label .edit-button.edit-button__active {
   opacity: 100%;
 }
+
+#assigneeInput {
+  position: relative;
+}
+
+#list-items {
+  width: 100%;
+  margin-top: 2rem;
+  position: absolute;
+  border: 1px solid #d4d4d4;
+  border-bottom: none;
+  border-top: none;
+  z-index: 99;
+  top: 50%;
+  left: 0%;
+  right: 0;
+  border-radius: 5px;
+  background-color: #fff;
+  box-shadow: 1px 2px 5px rgba(0, 0, 0, 0.2);
+}
+
+.list-item {
+  text-align: left;
+  padding: 1rem 0.5rem;
+  font-size: 1rem;
+  border-bottom: 1px solid #d4d4d4;
+  margin: 0;
+}
+
+#list-items:nth-last-child() {
+  border-bottom: 0;
+}
+
+.list-item:hover {
+  background-color: #ddd;
+}
+
+.user-list {
+  margin-top: 0.5rem;
```

Comment:
Having a smaller font size and less margin looks good, right? Also, let's make **User** as **user** and have margin on all the sides 
![image](https://user-images.githubusercontent.com/45538392/154098026-6b0e922c-214f-4762-91b5-5e4ac9da6315.png)

**margin : .2rem** would be good i guess

Reply:
Margin and fontsize changed